### PR TITLE
Add search

### PR DESCRIPTION
**Pull request by solomiyon**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by christykobyletska — APPROVED** [review]

**Review by OleksandrBilan — APPROVED** [review]

File: src/pages/Documents/DropDownDocuments.tsx [hunk 1]
```diff
@@ -36,22 +42,32 @@ const DropDown = (props: Props) => {
...
-        setClubAdm(roles.includes(Roles.KurinHead));
-      }
-      fetchData();
-    }
-  }, [showEditModal]);
+    const fetchData = async () => {
+      let jwt = AuthStore.getToken() as string;
+      let decodedJwt = jwt_decode(jwt) as any;
+      let roles = decodedJwt['http://schemas.microsoft.com/ws/2008/06/identity/claims/role'] as string[];
+      console.log(roles);
```
Comment (sharix4) [inline src/pages/Documents/DropDownDocuments.tsx:54]: need to be removed console.log(roles);

Comment (solomiyon) [reply src/pages/Documents/DropDownDocuments.tsx]: done

File: src/pages/Documents/DropDownDocuments.tsx [hunk 2]
```diff
@@ -26,6 +28,10 @@ const DropDown = (props: Props) => {
   const [regionAdm, setRegionAdm] = useState(false);
   const [cityAdm, setCityAdm] = useState(false);
   const [clubAdm, setClubAdm] = useState(false);
+  const [canEdit, setCanEdit] = useState(false);
+  const [userRole, setUser] = useState<string[]>();
```
[the remark this answers is not in the document]
Comment (solomiyon) [reply src/pages/Documents/DropDownDocuments.tsx]: done

**Review by sharix4 — APPROVED** [review]

**Review by andrii52 — APPROVED** [review]

File: src/pages/Documents/DocumentsTable.tsx [hunk 3]
```diff
@@ -175,23 +178,33 @@ const DocumentsTable = () => {
...
             tabList={tabList}
             activeTabKey={status}
             onTabChange={(key) => {
             onTabChange(key);
-            }}
-          >
+            }} 
+          /> ) : (
+          <Card
+              style={{ width: '100%' }}
```
Comment (danylo-melnychuk) [inline src/pages/Documents/DocumentsTable.tsx:203]: Are you sure that it's an adaptive Cart? 
Because you use  width: 100%, it's bad practise

Comment (solomiyon) [reply src/pages/Documents/DocumentsTable.tsx:203]: Yes, we can scroll it as in other pages

**Pull request merged**